### PR TITLE
issue-16 Use childNodes iso. element.children

### PR DESCRIPTION
**Pull request by krrm**

### CODE REVIEW

File: lib/childrensize.js [hunk 1]
```diff
@@ -10,13 +10,22 @@ var rectStore = null;
 function toArray (nodeList) {
     // Array.prototype.slice does not work on HTMLCollection in IE8
     var arr = [];
-    for (var i=0, l=nodeList.length; i<l; i++) { arr.push(nodeList[i]); }
+    if (nodeList) {
```
Comment (gregersrygg) [inline lib/childrensize.js]: I would prefer `if (!nodeList) { return []; }` to avoid unnecessary indentation.

File: lib/childrensize.js [hunk 2]
```diff
@@ -10,13 +10,22 @@ var rectStore = null;
...
+        var node;
+        for (var i=0, l=nodeList.length; i<l; i++) {
+            node = nodeList[i];
+            // Filter out anything but elements
```
Comment (gregersrygg) [inline lib/childrensize.js]: This function is for converting a nodeList to an array. Filtering should be done elsewhere.

**Comment by gregersrygg** [comment]
Related to #16 

Please squash commits before merging :innocent:

**Comment by sveisvei** [comment]
@krrm @gregersrygg is this ready to merge?

**Comment by krrm** [comment]
@sveisvei afaik yes

**Pull request merged**